src/site/content/en/fast/efficiently-load-third-party-javascript/index.md

```diff
@@ -83,15 +82,14 @@ Two [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) t
 
 ### `preconnect`
 
-`<link rel="preconnect">` informs the browser that your page intends to establish a connection to another origin, and that you'd like the process to start as soon as possible. When the request for a resource from the preconnected origin is made, the download starts immediately. 
+`<link rel="preconnect">` informs the browser that your page intends to establish a connection to another origin, and that you'd like the process to start as soon as possible. When the request for a resource from the pre-connected origin is made, the download starts immediately. 
```

Comment:
Idk, to shut up the spell checker I guess. I'll fix it.

Reply:
Thank you :) I'll add this to the word list too.

Reply:
While you're at it, add round trip too. That one's gonna need special effort to enforce, web fundamentals has a very diverse usage of round-trip/roundtrip/round trip.

Reply:
Is "round trip" definitely the preferred variant?

Reply:
Web fundamentals is fairly evenly split between round-trip and round trip I think. The networking bible published by O'Reilly (https://hpbn.co/) uses roundtrip. I think both condensed options are semantically more correct than the separated one. 

When you make a trip, it can be a _one way_ trip or a _round_ trip. In the context of networking, a sent request always receives a response (data, error or timeout) and that trip only ever exists as a round(-)trip.

Reply:
OK, given that argument, I'm going to propose "round trip" as a noun and "round-trip" as an adjective, which matches the typical convention in English. I'll run this by Addy.